examples/salty-ocean/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "my-gensx-app",
```

Comment:
This should be something like `@gensx-examples/salty-ocean` or `@gensx-examples/model-comparison`

Reply:
Updated!